integration-tests/tests/import-dockstore-workflow.js

```diff
@@ -23,9 +23,9 @@ const testImportDockstoreWorkflowFn = _.flow(
   withUserToken,
   withDockstoreCheck
 )(async ({ page, testUrl, token, workspaceName }) => {
-  await page.goto(`${testUrl}/#import-tool/dockstore/${testWorkflowIdentifier}`)
-  await signIntoTerra(page, token)
-  await dismissNotifications(page)
+  const url = `${testUrl}/#import-tool/dockstore/${testWorkflowIdentifier}`
+  await signIntoTerra(page, { token, testUrl: url })
```

Comment:
The usage here makes me think that `testUrl` is a misnomer in the parameter list. If you did:
```javascript
)(async ({ page, testUrl: testUrlRoot, token, workspaceName }) => {
```
then you could:
```suggestion
  const testUrl = `${testUrlRoot}/#import-tool/dockstore/${testWorkflowIdentifier}`
  await signIntoTerra(page, { token, testUrl })
```

Reply:
Excellent suggestion.